### PR TITLE
mongodb-atlas-cli 1.31.0

### PR DESCRIPTION
**Pull request by dnhn**
Created by `brew bump`

-----

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/mongodb-atlas-cli.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class MongodbAtlasCli < Formula
   desc "Atlas CLI enables you to manage your MongoDB Atlas"
   homepage "https://www.mongodb.com/docs/atlas/cli/stable/"
-  url "https://github.com/mongodb/mongodb-atlas-cli/archive/refs/tags/atlascli/v1.10.0.tar.gz"
-  sha256 "d005e51d610526c7dec4623ce4c9f2975f7c0d96a7f1548cd8f5f36188cf08c0"
+  url "https://github.com/mongodb/mongodb-atlas-cli/archive/refs/tags/mongocli/v1.31.0.tar.gz"
```
Comment (chenrui333) [inline Formula/mongodb-atlas-cli.rb:4]: this is wrong, this is for mongocli

Comment (chenrui333) [reply Formula/mongodb-atlas-cli.rb:4]: the livecheck would also report error

```
$ brew livecheck mongodb-atlas-cli
mongodb-atlas-cli: 1.10.0 ==> 1.10.0
```

Comment (chenrui333) [reply Formula/mongodb-atlas-cli.rb:4]: see https://github.com/Homebrew/homebrew-core/pull/131959

Comment (chenrui333) [reply Formula/mongodb-atlas-cli.rb:4]: - actually this, #139262

you can use `brew bump --open-pr mongocli` next time

Comment (dnhn) [reply Formula/mongodb-atlas-cli.rb:4]: Thank you.

**Pull request closed without merging**